shims/spark300/src/main/scala/com/nvidia/spark/rapids/shims/spark300/Spark300Shims.scala

```diff
@@ -134,6 +135,10 @@ class Spark300Shims extends SparkShims {
   override def isShuffleExchangeLike(plan: SparkPlan): Boolean =
     plan.isInstanceOf[ShuffleExchangeExec]
 
+  override def getQueryStageRuntimeStatistics(plan: QueryStageExec): Statistics = {
+    Statistics(0, None)
```

Comment:
nit: `Statistics(0)` since None is the default for rowcount

Reply:
Fixed